code/src/questions.json

```diff
@@ -0,0 +1,62 @@
+{
+    "nameQuestion": "What is your name?",
+
+    "durationQuestion": "How long were we together?",
+
+    "duration": [
+        "0-3 months",
+        "3-6 months",
+        "6-12 months",
+        "1-1.5 years",
+        "1.5-2 years",
+        "2-5 years",
+        "5-10 years",
+        "10+ years"
+    ],
+
+    "breakupQuestion": "Who broke up with whom?",
+
+    "breakupAlts": [
+        "I broke up with you.",
+        "You broke up with me.",
+        "It was mutual.",
+        "No, it really was mutual!"
+    ],
+
+    "breakupReasonQuestion": "Why do you think we broke up?",
+
+    "traitsQuestion": "What are some of my worst charachter traits?",
+
+    "traits": [
+        "lazy",
+        "late",
+        "boring",
+        "ugly",
+        "stubborn",
+        "loud",
+        "mean",
+        "petty",
+        "vain",
+        "selfish",
+        "self-indulgent",
+        "greedy",
+        "narcissistic",
+        "judgmental",
+        "spoiled",
+        "needing to be right",
+        "All of the above"
+    ],
+
+    "backTogetherQuestion": "Would you ever consider getting back together with me?",
+
+    "backTogetherAlts": [
+        "Absolutely!",
+        "Never in a million years!",
+        "Nah.",
+        "Maybe?"
+    ],
+
+    "dateFriendQuestion": "Would you let one of your friends date me?",
+
+    "friendNameQuestion": "Who???"
+}

```

Comment:
OMG SO SMART PUTTING ALL YOUR QUESTIONS IN A JSON FILE.... AHHH THIS IS SO SMART AND ADVANCED